demo/JENKINS_HOME/jobs/docker-workflow/config.xml

```diff
@@ -50,13 +53,17 @@ docker.withRegistry(&apos;https://docker.example.com/&apos;, &apos;docker-regist
     stage &apos;Test Image&apos;
     // Run the petclinic app in its own docker container
     pcImg.withRun {petclinic -&gt;
-        // Spin up a maven test container, linking it to the petclinic app container allowing
-        // the maven tests to fire HTTP requests between the containers.
-        maven32.inside(&quot;-v /m2repo:/m2repo --link=${petclinic.id}:petclinic&quot;) {
-            git &apos;https://github.com/tfennelly/spring-petclinic-tests.git&apos;
+        dir(&apos;test&apos;) {
+        git url: &apos;https://github.com/jglick/spring-petclinic-tests.git&apos;, branch: &apos;selenium&apos;
```

Comment:
[the diff](https://github.com/tfennelly/spring-petclinic-tests/compare/master...jglick:selenium)